indico_toolkit/indico_wrapper/modelop.py

```diff
@@ -0,0 +1,204 @@
+from indico import IndicoClient
+from indico.queries import GraphQLRequest
+import json
+import string
+
+
+class ModelOp:
+
+    """
+    Class to assit in Model Group Operation calls.
+    """
+
+    def __init__(self, client: IndicoClient):
+        self.client = client
+        self.task_type = None
+
+    def get_model_group(self, model_group_id: int) -> dict: 
+        """
+        Get specified model group information.
+        Args:
+            model_group_id (int): id of model group 
+        Returns:
+            dict: attributes of the specified model group id 
+        """
+
+        query = """
+        
+query GetModelGroup($id: Int) {
```

Comment:
You'll want to run black to format this whole doc-- but in this case, I'd eliminate the whitespace on line 27 and indent this (one tab in from query)